src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -23,16 +23,42 @@ databaseChangeLog:
                   type: varchar(255)
                   constraints:
                     nullable: false
-  - changeSet:
-      id: 2
-      author: daniel-kujawa
+              - column:
+                  name: role
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+ 
       changes:
-        - insert:
-            tableName: users
+        - createTable:
+            tableName: tickets
             columns:
               - column:
-                  name: username
-                  value: "user"
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
               - column:
-                  name: password
-                  value: "pass"
\ No newline at end of file
+                  name: title
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: description
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                   name: issuer_id
```

Comment:
For me, issuer_id and tester_id make no sense. Tester IS the issuer in our project. I suggest to change the names of these 2 columns to author_id and developer_id.

Reply:
Done as suggested.